includes/class-sensei-content-drip.php

```diff
@@ -415,4 +415,17 @@ public function load_template( $name, $args = array() ) {
 		$theme_path   = 'sensei-content-drip/';
 		Sensei_Templates::get_template( $name, $args, $theme_path, $default_path );
 	}
+
+	/**
+	 * Check if course enrolment is handled by the legacy method.
+	 *
+	 * @since 2.0.2
+	 */
+	public function is_legancy_enrolment() {
```

Comment:
Typo on the method name "lega**n**cy".

Reply:
Fixed in ea32108b0257905a88d5e6f24e760f16fded879c